src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -0,0 +1,65 @@
+<%--
+  Copyright 2017 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--%>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Git Rekt's Chat App</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <nav>
+    <a id="navTitle" href="/">Git Rekt's Chat App</a>
+    <a href="/conversations">Conversations</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
+      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else{ %>
+      <a href="/login">Login</a>
+    <% } %>
+    <a href="/about.jsp">About</a>
+  </nav>
+
+  <div id="container">
+    <div
+      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+      <% 
+       String user = (String) request.getSession().getAttribute("user"); 
+       String author = (String) request.getAttribute("author"); 
+       boolean match = (Boolean) request.getAttribute("userMatch");
+       if (match) { %> 
```

Comment:
Could you refactor this part and put the "author"'s Profile and the About me and their info regardless of whether or not its their own profile? The code here is duplicated in lines 56-59 and it looks like you might be able to move it outside and just use the author attribute before you even check for any match

Reply:
No problem, I know the lines you are talking about haha. The way I had done it I had used the user's name for one and the author's name for the other, but I went ahead and changed it so that the page only uses the author's name or else the page might display null's Profile if the user is not logged in.